### PR TITLE
Style footer 

### PR DESCRIPTION
**Pull request by Precoder365**

### CODE REVIEW

**Comment by AbhijithGanesh** [comment]
@Precoder365 Just a suggestion, next time donot push from master

File: src/components/footer.tsx [hunk 1]
```diff
@@ -38,21 +35,21 @@ const IconKnowMore = (props: KnowMoreProps) => {
 const Footer = (): JSX.Element => {
   return (
     <>
-      <footer className="border border-t-[1px] border-b-0 border-r-0 border-l-0 border-textcolors-boundary bg-darkshades-passive py-2 font-Plex">
-        <section className="grid sm:grid-cols-1 md:grid-cols-6 md:gap-2 lg:grid-cols-10">
-          <section className=" hidden items-center justify-start md:col-span-2 md:col-start-1 md:flex lg:col-span-3 lg:col-start-1 lg:flex">
+      <footer className="border border-t-[2px] border-b-0 border-r-0 border-l-0 border-textcolors-boundary bg-darkshades-passive py-2 font-Plex content-center">
+        <section className="p-7 flex flex-row">
```
Comment (AbhijithGanesh) [inline src/components/footer.tsx:39]: Just try centring the element. 
```suggestion
        <section className="p-7 flex flex-row items-center">
```

**Review by AbhijithGanesh — APPROVED** [review]

**Comment by Precoder365** [comment]
> @Precoder365 Just a suggestion, next time donot push from master

Noted

File: src/components/footer.tsx (hunk 1 above)
Comment (Precoder365) [reply src/components/footer.tsx]: Yes done.

**Pull request merged**